controllers/edit_profile.php

```diff
@@ -1,3 +1,50 @@
 <?php
+function loopGetinfo($sql)
+{
+    foreach ($sql as $statementresults => $statementresult) {
+        foreach ($statementresult as $results) {
+            return $results;
+        }
+    }
+}
+function updateUserInfo()
+{
+    return [
+        "firstName" => loopGetinfo(customStatement("SELECT first_name FROM users WHERE " . $_SESSION['users_id'] . " = users.users_id")),
+        "lastName" => loopGetinfo(customStatement("SELECT last_name FROM users WHERE " . $_SESSION['users_id'] . " = users.users_id")),
+        "email" => loopGetinfo(customStatement("SELECT email FROM users WHERE " . $_SESSION['users_id'] . " = users.users_id")),
+    ];
+}
```

Comment:
1. `updateUserInfo` moet zijn `getUserInfo` want je doet hier geen update maar haalt gegeven op.

2. Je kan alle gegevens in 1 keer ophalen in een select query. Bijvoorbeeld met een ster (*) of columnNamen specifiek. 
De methode `customStatement` gebruikt een fetch(PDO:FETCH_ASSOC) waardoor je een array terug krijgt met de juiste columnNamen. 
De WHERE moet eerst de columnname hebben en daarna de waarde. `WHERE users_id = $_SESSION['users_id']`

Zie hier een code voorbeeld waar je exact hetzelfde kan bereiken: (met beveiliging)
```
$sql = "SELECT * from `users` WHERE users_id = :user_id";
$stmt = $conn->conn->prepare($sql);
$stmt->bindParam(':user_id', $_SESSION['users_id']);
$stmt->execute();
return $stmt->fetchAll(PDO::FETCH_ASSOC);
```

Reply:
https://stackoverflow.com/a/16846612

En dan loop je eroverheen met:
```
foreach ($users as $user) {
    $user['first_name'];
    $user['last_name'];
}
```

Reply:
ik ga deze morgen nog even uitzoeken want ik kom er nu niet helemaal uit

Reply:
Dan moet ik niet de customstatement functie gaan gebruiken toch?

Reply:
als je die code overneemt niet nee